device.c

```diff
@@ -156,6 +158,12 @@ struct device *device_open(const char *board,
 	if (!device->console)
 		errx(1, "failed to open device console");
 
+	/* Power off before opening fastboot */
```

Comment:
off?

Reply:
Yes. We need to power it off before starting the fastboot detection, otherwise the device might get stuck in the reset-second_fastboot-quit cycle, if it was in the fastboot mode during open.

Reply:
put that in the comment, please

Reply:
done